econml/inference.py

```diff
@@ -45,26 +45,69 @@ class BootstrapInference(Inference):
     n_jobs: int, optional (default -1)
         The maximum number of concurrently running jobs, as in joblib.Parallel.
 
+    bootstrap_type: 'percentile', 'pivot', or 'normal', default 'percentile'
+        Bootstrap method used to compute results.
+        'percentile' will result in using the empiracal CDF of the replicated copmutations of the statistics.
+        'pivot' will also use the replicates but create a pivot interval that also relies on the estimate
+        over the entire dataset.
+        'normal' will instead compute a pivot interval assuming the replicates are normally distributed.
     """
 
-    def __init__(self, n_bootstrap_samples=100, n_jobs=-1):
+    def __init__(self, n_bootstrap_samples=100, n_jobs=-1, bootstrap_type='percentile'):
```

Comment:
I think default should be pivot.

Reply:
That would be better, but is not backwards compatible.  Since it's not a big behavior change (and it's almost certainly an improvement), I'm happy to make the change if you think that's more important.

Reply:
Yeah I would switch to pivot as it's a better approach.

Reply:
This will be fixed in an update I'm about to push.